networking/network_observability/network-observability-overview.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id='network-observability-overview']
+= About Network Observability
+include::_attributes/common-attributes.adoc[]
+:context: network-observability-overview
+
+toc::[]
+
+Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters. The NetObserv eBPF agent uses eBPF technology to create network flows based on the IPFIX format. Alternatively, you can export network flows if you are using OVN Kubernetes, but this is not supported.  The network flow data is saved to an object store. You can access the network flow through Loki by using Web console or Grafana
+
+[id="dependency-network-observability"]
+== Dependency of Network Observability Operator
+
+The Network Observability Operator requires the following Operators:
+
+* Loki Operator: It is recommended to install the Loki Operator for the installation of Network Observability Operator. The Loki Operator is the backend that you can use to store all collected flows.
+
+[id="optional-dependency-network-observability"]
+== Optional dependency of Network Observability Operator
+
+* Grafana Operator: You can install Grafana Operator for using custom dashboards and querying capabilities. Red Hat does not support Grafana Operator.
+* Kafka Operator: It is recommended to install Kafka using AMQ Streams. This Operator provides scalability, resiliency and high availability in the {product-title} cluster.
```

Comment:
I would lead with why kafka, and then the recommendation to use AMQ streams.
So just switch the sentence order.
It might be good to mention that it's intended for larger scale deployments/etc.

Reply:
Addressed